geometric_controller/include/geometric_controller/common.h

```diff
@@ -46,6 +46,17 @@
 #include <geometry_msgs/TwistStamped.h>
 #include <Eigen/Dense>
 
+const size_t nStates = 10;
+const size_t nControls = 4;
+
+typedef Eigen::Matrix<double, nStates, 1> state_vector_t;
+typedef Eigen::Matrix<double, nControls, 1> control_vector_t;
+
+typedef Eigen::Matrix<double, nStates, nStates> state_matrix_t;
+typedef Eigen::Matrix<double, nControls, nControls> control_matrix_t;
+typedef Eigen::Matrix<double, nStates, nControls> control_gain_matrix_t;
```

Comment:
I don't think common is the right place for these typedefs. This should belong to the controller since these are not shared with any other controller

Reply:
Makes sense. I can declare on the geometric controller class.